config/locales/plurals.rb

```diff
@@ -1,34 +1,49 @@
+# frozen_string_literal: true
```

Comment:
This commit has a definition for `[1]`, but `[1]` isn't reference anywhere 😁

Reply:
Lol - have now referenced it correctly 😄